qualtran/bloqs/reflections/reflection_using_prepare_test.py

```diff
@@ -133,7 +133,7 @@ def test_reflection_using_prepare(num_ones, eps, global_phase):
     prepared_state = result.final_state_vector.reshape(2 ** len(selection), -1).sum(axis=1)
     if np.sign(global_phase) == 1:
         signs = '-' * num_ones + '+' * (9 - num_ones)
-    elif np.sign(global_phase) == -1:
+    else:  # np.sign(global_phase) == -1:
```

Comment:
can we keep this as an elif and add an else clause that raises a ValueError or AssertionError?